migrator/migrations/m_209_to_m_210_add_updated_at_to_network_flows_v2/migration_impl.go

```diff
@@ -0,0 +1,53 @@
+package m209tom210
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/migrator/migrations/loghelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+var (
+	log = loghelper.LogWrapper{}
+)
+
+func migrate(database *types.Databases) error {
+	ctx := sac.WithAllAccess(context.Background())
+	tableName := "network_flows_v2"
+	column := "updatedat"
+	if err := addColumnToTable(ctx, database.PostgresDB, tableName, column); err != nil {
+		log.WriteToStderrf("unable to alter table %s: %v", tableName, err)
+		return err
+	}
+
+	return nil
+}
+
+func addColumnToTable(ctx context.Context, db postgres.DB, table, column string) error {
+	ctx, cancel := context.WithTimeout(ctx, types.DefaultMigrationTimeout)
+	defer cancel()
+	alterTableStmt := fmt.Sprintf("alter table %s add column if not exists %s timestamp without time zone;", table, column)
```

Comment:
nit:  use caps on SQL key words as is typically done.

Reply:
Done